automatic/git.install/tools/helpers.ps1

```diff
@@ -0,0 +1,69 @@
+function Get-InstallKey() {
+    $registryKeyName = 'Git_is1'
+    $installKey = "HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\$registryKeyName"
+    if ((Get-ProcessorBits 64) -and $env:chocolateyForceX86 -ne 'true') {
```

Comment:
this looks wrong to me, basically you're saying if we are running on 64bit and 32bit install isn't forced, use the Wow6432Node registry location.
I believe it should be:
```
(Get-ProcessorBits 64) -and $env:chocolateyForceX86 -eq 'true'
```